genref/html/pkg.tpl

```diff
@@ -0,0 +1,73 @@
+{{ define "packages" }}
+  <html lang="en">
+    <head>
+      <meta charset="utf-8">
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.8.2/css/font-awesome.min.css">
+      <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+      <style type="text/css">
+        td p {
+          margin-bottom: 0
+        }
+        code {
+          color: #802060;
```

Comment:
Do you want to leave this style element inline or add these attributes to the site's .scss file(s)?

Reply:
This doesn't matter because we will not publish HTML output to the website.